src/components/Main-nav/MainNav.scss

```diff
@@ -0,0 +1,260 @@
+@import '../shared/main.scss';
+
+.main-menu{
+	&__nav{
+		width: 100%;
+		height: 5vh;
+		display: flex;
+		cursor: pointer;
+		font-family: $font-family;
+
+		@include large{
+			width: 100%;
+			height: 100px;
+			position: relative;
+			background: -webkit-linear-gradient(left, $primaryColor, $secondaryColor);
```

Comment:
You don't need to use prefixed css as the `-webkit-linear-gradien`. If it is really necessary, add a new plugin in the Webpack configuration to help you with that. If you need help with this, please let me know.